Rules

```diff
@@ -34,6 +34,9 @@ end
 compile '/static/**/*' do
 end
 
+compile '/authorization/cert_example.png' do
+end
+
```

Comment:
This and the authorization file changes were temporary changes I had to make to get the website to compile locally. Hopefully, we will get some input on how to compile without these.